resolwe_bio/processes/expression/featureCounts.yml

```diff
@@ -4,112 +4,346 @@
 ---
 
 - slug: feature_counts
-  name: FeatureCounts
+  name: featureCounts
   requirements:
     expression-engine: jinja
     executor:
       docker:
-        image: resolwebio/legacy:latest
-  data_name: "{{ alignments.bam.file|basename|default('?') }} ({{ (alignments|sample_name) }})"
-  version: 0.3.2
+        image: resolwebio/rnaseq:2.1.1
+    resources:
+      cores: 1
```

Comment:
Can you do a quick performance testing on one regular-sized BAM file with 2, 4 and 8 cores to see the level of improvement we might get? I suggest to do it in terminal on the local machine.